### PR TITLE
chore(tabs): add paparazzi tests

### LINKED ISSUE #96: test(tab): Add paparazzi tests
**Describe the solution you'd like**
Write tests with paparazzi for tab component for all variants & themes. 

**Additional context**
FYI Buttons already have ones.

### PR DESCRIPTION
**Pull request by FDELAHA24**
## Changes description 🧑‍💻

Add paparazzi tests for tabs 

## Context 🤔

Closes #96 

## Checklist ✅

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [x] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [x] Check compose contract convention. It must follow conventions described [here](/CONVENTIONS.md).
- [x] Check your code additions will fail neither code linting checks.
- [x] I have reviewed the submitted code.
- [x] I have tested on a phone device/emulator.
- [ ] I have tested on a tablet device/emulator.
- [ ] I have tested on a large screen device/emulator.
- [ ] If it includes design changes, please ask for a review `design-system-core-team-design` GitHub team.

### CODE REVIEW

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/e9ae480151be4514/build/bc08fbf3-94de-4c44-a75c-3c76b8897872/artifact/082b7f83ce04ef53/p/01984fdb970bb0efd02b2c595afd4a1c

File: tabs/src/test/kotlin/com/decathlon/vitamin/compose/tabs/VitaminButtonsPrimaryTest.kt [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import com.decathlon.vitamin.compose.tabs.utils.Variant
+import com.google.testing.junit.testparameterinjector.TestParameter
+import com.google.testing.junit.testparameterinjector.TestParameterInjector
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+
+@RunWith(TestParameterInjector::class)
+class VitaminButtonsPrimaryTest(
```
Comment (ManonPolle) [inline tabs/src/test/kotlin/com/decathlon/vitamin/compose/tabs/VitaminButtonsPrimaryTest.kt:23]:
```suggestion
class VitaminTabsTest(
```

**Review by ManonPolle — CHANGES_REQUESTED** [review]

**Comment by dktbitrise** [comment]
### You can download the apk here :

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/e9ae480151be4514/build/808e7afa-a138-42bc-b13c-8e1fb87fecf7/artifact/2e0f5c318d4f7ed1/p/5780eebca18895767fa2c560eb9f0183

**Review by ManonPolle — APPROVED** [review]

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/e9ae480151be4514/build/1e06a445-18d3-4c03-ab5b-f3ceaff78751/artifact/d975043c775c0fd2/p/2bddd96d4425caa428a1e32d29e9dd7a

**Pull request merged**